pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -422,10 +422,10 @@ func Test_EnforceCommandPolicy_NarrowingMatches(t *testing.T) {
 		// create two additional containers that "share everything"
 		// except that they have different commands
 		testContainerOne := generateContainersContainer(testRand, 5)
-		testContainerTwo := testContainerOne
+		var testContainerTwo securityPolicyContainer = *testContainerOne
```

Comment:
nit: `testContainerTwo := *testContainerOne`

Reply:
I agree with the nit! I just submitted an update to this PR.